libraries/browser-tracker-core/src/tracker/out_queue.ts

```diff
@@ -137,16 +144,20 @@ export function OutQueueManager(
   }
 
   // Used by pageUnloadGuard
-  sharedSate.outQueues.push(outQueue);
+  sharedState.outQueues.push(outQueue);
 
   if (useXhr && bufferSize > 1) {
-    sharedSate.bufferFlushers.push(function (sync) {
+    sharedState.bufferFlushers.push(function (sync) {
       if (!executingQueue) {
         executeQueue(sync);
       }
     });
   }
 
+  function getQueueName(id: string, method: Exclude<EventMethod, 'beacon'>) {
+    return `snowplowOutQueue_${id}_${method === 'post' ? 'post2' : 'get'}`;
```

Comment:
Since the type isn't checked at runtime and POST is the default these days, should we flip this and check for GET instead, defaulting to post?

So even though `getQueueName('beacon')` isn't actually correct use of the API, it'll still produce the correct/desired result more often? Probably a philosophical thing though, [POLA](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) vs [GIGO](https://en.wikipedia.org/wiki/Garbage_in,_garbage_out).

Reply:
Sure, good idea :)